### PR TITLE
Icdc facet filter

### PR DESCRIPTION
**Pull request by rana22**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: packages/facet-filter/README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ import FacetFilter from 'bento-core';
     facetsConfig={facetsConfig}
     CustomFacetSection={CustomFacetSection}
     CustomFacetView={CustomFacetView}
+    enableClearSection // add additional clear button on facet Section
```
Comment (iksheth) [inline packages/facet-filter/README.md:57]: Hi @rana22, Is this one enabled in bento-RI by default or Out of the box for any new project that are cloning Bento?

File: packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js [hunk 2]
```diff
@@ -5,6 +5,8 @@
 /* eslint-disable object-curly-newline */
```
Comment (iksheth) [inline packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js:5]: @rana22 Can we avoid es-lint-disable? If possible can we remove some as well if those are quick fix.

File: packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js [hunk 3]
```diff
@@ -60,9 +66,7 @@ const CheckBoxView = ({
         [`${checkedSection}NameChecked`]: isChecked,
       })}
     >
-      {label ? (
-        <Typography className={classes.checkboxLabel}>{label}</Typography>
-      ) : (<Typography className={classes.checkboxName}>{name}</Typography>)}
+      <Typography className={classes.checkboxName}>{name}</Typography>
```
Comment (iksheth) [inline packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js:69]: @rana22, Looks like "label" is missing is that intentional or mistake? if intentional can you explain rational behind it?

**Review by iksheth — CHANGES_REQUESTED** [review]

File: packages/facet-filter/README.md (hunk 1 above)
Comment (rana22) [reply packages/facet-filter/README.md:57]: It should be configured from the projects.

File: packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js (hunk 2 above)
Comment (rana22) [reply packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js]: Removed

File: packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js (hunk 3 above)
Comment (rana22) [reply packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js:67]: The label and name variables are redundant so only one is used.

**Review by iksheth — APPROVED** [review]

**Pull request merged**